nautobot_circuit_maintenance/__init__.py

```diff
@@ -82,5 +82,11 @@ def ready(self):
         post_migrate.connect(custom_fields_extension, sender=self)
         post_migrate.connect(import_notification_sources, sender=self)
 
+        # pylint: disable=import-outside-toplevel
+        from nautobot_capacity_metrics import register_metric_func
+        from .metrics_app import metric_active_maintenances
+
+        register_metric_func(metric_active_maintenances)
```

Comment:
I think it would be good to have a flag in the config of the plugin to enable / disable this feature if needed